File: C++/4. Median of Two Sorted Arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& n1, vector<int>& n2) {
        int i,n;
        double p;
        for(i=0;i<n2.size();i++)
        {
            n1.push_back(n2[i]);
        }
        sort(n1.begin(),n1.end());
        n=n1.size();
        if(n%2==0)
        {
            n=n/2;
            p=n1[n-1]+n1[n];
            return(p/2);
        }
        else
        {
            n=n/2;
            p=n1[n];
            return(p);
        }

    }
};

File: JAVA/4. Median of Two Sorted Arrays.java
class Solution {
    public double findMedianSortedArrays(int[] n1, int[] n2) {
        int i;
        double m;
        int[] n=new int[n1.length+n2.length];
        for(i=0;i<n1.length;i++){
            n[i]=n1[i];
        }
        for(i=0;i<n2.length;i++){
            n[i+n1.length]=n2[i];
        }
        Arrays.sort(n);
        if(n.length%2==0)
        m=(n[n.length/2]+n[n.length/2-1])/2.0;
        else
        m=n[n.length/2];
        return m;
    }
}